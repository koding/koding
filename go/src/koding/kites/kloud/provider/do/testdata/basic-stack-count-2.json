{
  "provider": {
    "digitalocean": {
      "access_token": "${var.digitalocean_access_token}"
    }
  },
  "resource": {
    "digitalocean_droplet": {
      "do-instance": {
        "count": 2,
        "image": "ubuntu-14-04-x64",
        "name": "koding-${var.koding_group_slug}-${var.koding_stack_id}-${count.index+1}",
        "region": "nyc2",
        "size": "512mb",
        "user_data": "\\necho \\\"hello world!\\\" >> /helloworld.txt\\n"
      }
    }
  }
}
