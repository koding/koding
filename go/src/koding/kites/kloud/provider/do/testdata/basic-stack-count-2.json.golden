{
  "provider": {
    "digitalocean": {
      "token": "12345"
    }
  },
  "resource": {
    "digitalocean_droplet": {
      "do-instance": {
        "count": 2,
        "image": "ubuntu-14-04-x64",
        "name": "...",
        "region": "nyc2",
        "size": "512mb",
        "ssh_keys": [
          56789
        ],
        "user_data": "..."
      }
    },
    "null_resource": {
      "do-instance": {
        "depends_on": [],
        "triggers": {
          "user_data": "\\necho \\\"hello world!\\\" >> /helloworld.txt\\n"
        }
      }
    }
  },
  "variable": {
    "kitekeys_do-instance": {
      "default": {
        "0": "...",
        "1": "..."
      }
    }
  }
}
